config/csm/csm_config.yaml

```diff
@@ -26,6 +26,8 @@ Restcall:
   s3_bucket_endpoint : "/api/v2/s3/bucket"
   IAM_users_endpoint : "/api/v2/iam_users"
   s3_iam_user_endpoint : "/api/v2/s3/iam/users"
+  s3_iam_keys_endpoint : "/api/v2/s3/iam/keys"
+  s3_iam_caps_endpoint : "/api/v2/s3/iam/caps"
```

Comment:
This endpoints are not applicable for LR, So please remove them from here and use it from Restcall_LC

Reply:
Resolved.